stock_picking_group_by_partner_by_carrier/models/stock_move.py

```diff
@@ -57,6 +57,7 @@ def _domain_search_picking_for_assignation(self):
         domain = super()._domain_search_picking_for_assignation()
         if (
             not self.picking_type_id.group_pickings
+            or self.partner_id.disable_picking_grouping
```

Comment:
_is_grouping_disabled() ?

Reply:
In this method (stock.move level) we have no picking yet (it's the purpose of the method to find one), so we can't use the `_is_grouping_disabled` method of the picking here. We could implement one for the move maybe but it's only used here.